perfkitbenchmarker/providers/yandexcloud/util.py

```diff
@@ -0,0 +1,213 @@
+# Copyright 2020 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utilities for working with Yandex.Cloud resources."""
+
+import collections
+import json
+import yaml
+import logging
+import re
+from perfkitbenchmarker import context
+from perfkitbenchmarker import errors
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import virtual_machine
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.providers.yandexcloud import yc_machine_types
+import six
+
+if six.PY2:
+  import functools32 as functools
+else:
+  import functools
+
+FLAGS = flags.FLAGS
+FLAVORS = yc_machine_types.FLAVORS
+
+
+@functools.lru_cache()
+def GetDefaultFolderId():
+  """Get the default folder id."""
+  conf = ParseConfig()
+  if "folder-id" in conf:
```

Comment:
Nit: you can say `return conf.get('folder-id')` to return None if the key is not in there.